### PR TITLE
oca-create-branch: carry over answers from previous branches

### PR DESCRIPTION
**Pull request by sbidoul**
Carry over the repo name and repo description from previous branch.

### CODE REVIEW

File: tools/create_branch.py [hunk 1]
```diff
@@ -25,7 +48,11 @@
     "repos",
     multiple=True,
 )
-def main(new_branch, copier_template, copier_template_vcs_ref, repos):
+@click.option(
+    "--prev-branch",
+    help="Previous branch where to read some copier answers.",
```
Comment (simahawk) [inline tools/create_branch.py:53]: default True?

**Review by simahawk — COMMENTED** [review]
LG, just a question

File: tools/create_branch.py (hunk 1 above)
Comment (sbidoul) [reply tools/create_branch.py:49]: It is actually a branch name.

**Comment by simahawk** [comment]
@sbidoul in the repo-maintainer tool will already preserve the repo name and description. Hence, I don't think we need this anymore.
See https://github.com/OCA/repo-maintainer/blob/master/oca_repo_maintainer/tools/manager.py#L191

Am I wrong?

Then we might implement a way to keep other keys but it can be done later.

**Comment by sbidoul** [comment]
I think with repo_description I wanted to preserve the body of the repo README.md. But in practice very few repo have it in a copier answer.

So happy to drop oca-create-branch from here and use repo-maintainer-conf for that.

Thanks for that work!

**Comment by sbidoul** [comment]
I keep this open for a while and if we are happy with the new solution, I'll remove create_branch.py from this repo.

**Comment by sbidoul** [comment]
Closing since this is now handled by github.com/oca/repo-maintainer-conf

**Pull request closed without merging**